newIDE/app/src/EventsSheet/InlinePopover.js

```diff
@@ -54,6 +54,7 @@ export default function InlinePopover(props: Props) {
 
   return (
     <ClickAwayListener
+      mouseEvent="onMouseDown"
```

Comment:
I can see in the comment below that this listener is also meant to work for mobile, can you check it works well on touchScreens/mobile?

Reply:
Yes, it works well. There's a dedicated props for that: `touchEvent`. I tried things on mobile but nothing changed really.